b2sdk/transfer/outbound/copy_manager.py

```diff
@@ -216,11 +215,14 @@ def _copy_small_file(
                 legal_hold=legal_hold,
                 file_retention=file_retention,
             )
-            file_info = FileVersionFactory.from_api_response(response)
+            file_version = self.services.api.file_version_factory().from_api_response(
+                self.services.api,
+                response,
+            )
             if progress_listener is not None:
-                progress_listener.bytes_completed(file_info.size)
+                progress_listener.bytes_completed(file_version.size)
 
-        return file_info
+        return file_version
```

Comment:
file_info, file_version, file_version_info... I am happy that we are fixing it and we will remove all such errors :)